File: lab3KononKI305/src/Messaging.java
// Messaging.java
import java.util.HashMap;
import java.util.Map;

public class Messaging {
    private Map<String, String> messages;

    // Default constructor
    public Messaging() {
        messages = new HashMap<>();
    }

    // Add message to the map
    public void addMessage(String phoneNumber, String message) {
        messages.put(phoneNumber, message);
        System.out.println("Message sent to " + phoneNumber + ": " + message);
    }

    // Display all messages
    public void displayMessages() {
        System.out.println("Messages:");
        for (Map.Entry<String, String> entry : messages.entrySet()) {
            System.out.println("To: " + entry.getKey() + ", Message: " + entry.getValue());
        }
    }
}

File: lab9KononKI305/messaging.py
# messaging.py
class Messaging:
    def __init__(self):
        self.messages = {}

    def add_message(self, phone_number, message):
        self.messages[phone_number] = message
        print("Message sent to", phone_number + ":", message)

    def display_messages(self):
        print("Messages:")
        for phone_number, message in self.messages.items():
            print("To:", phone_number + ",", "Message:", message)
